l4rm4nd_VoucherVault_myapp_views_59.py

```
@@ -0,0 +1,775 @@
+    # Count the number of items shared by the user
+    shared_items_count_by_you = ItemShare.objects.filter(shared_by=user).values('item').distinct().count()
+    shared_items_count_with_you = ItemShare.objects.filter(
+        shared_with_user=user,
+        item__is_used=False,
+        item__expiry_date__gte=now().date()
+    ).exclude(item__user=user).values('item').distinct().count()
+
+    # Get threshold days from environment variable or default to 30
+    threshold_days = int(os.getenv('EXPIRY_THRESHOLD_DAYS', 30))
+    # Calculate soon-to-expire date
+    soon_expiry_date = now() + timedelta(days=threshold_days)
```

Ruff (0.13.1)

`timedelta` may be undefined, or defined from star imports

(F405)